docs/admin/notifications.md

```diff
@@ -0,0 +1,293 @@
+# Notifications
+
+Notifications are sent by Coder in response to specific internal events, such as
+a workspace being deleted or a user being created.
+
+**Notifications are currently an experimental feature (as of their release in
+[2.15.0](https://github.com/coder/coder/releases/tag/v2.15.0)).**
+
+## Enable experiment
+
+In order to activate the notifications feature, you'll need to enable the
+`notifications` experiment.
+
+```bash
+# Using the CLI flag
+$ coder server --experiments=notifications
+
+# Alternatively, using the `CODER_EXPERIMENTS` environment variable
+$ CODER_EXPERIMENTS=notifications coder server
+```
+
+More information on experiments can be found
+[here](https://coder.com/docs/contributing/feature-stages#experimental-features).
+
+## Event Types
+
+Notifications are sent in response to internal events, to alert the affected
+user(s) of this event. Currently we support the following list of events:
+
+### Workspace Events
```

Comment:
Should we mention them here? Can we link to the code so it won't get outdated?

Reply:
I think it's ok to be honest; I would rather these be stale than force readers to read code (which they may not be able to do) to understand which events are considered.

Reply:
Ok